.github/workflows/build.yml

```diff
@@ -320,6 +326,103 @@ jobs:
           name: build-logs-winui${{ matrix.winui }}
           path: ./*.*log
 
+  sign:
+    needs: [package]
+    if: ${{ github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/heads/rel/') }}
```

Comment:
For Labs we should only sign the release packages we'll make once a week, for everything else they can remain as-is, don't need to take that extra time.

Reply:
Fixed in 58ac7474833886e090d07f04584fedfb917e1776